indico/modules/events/contributions/models/persons.py

```diff
@@ -60,6 +61,10 @@ class ContributionPersonLink(PersonLinkBase):
     def is_submitter(self):
         return self.person.has_role('submit', self.contribution)
 
+    @locator_property
+    def locator(self):
+        return dict(self.contribution.locator, author_id=self.person_id)
```

Comment:
`person_id` would make more sense IMO since it's not the id of the link but of the person